plugins/azure/applicationGateway/agMaxRequestBodySize.js

```diff
@@ -0,0 +1,66 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Application Gateway Max Request Body',
+    category: 'Application Gateway',
+    domain: 'Network Access Control',
+    description: 'Ensures that Application Gateway WAF policy have desired request body size configured.',
+    more_info: 'Application Gateway WAF policy includes a maximum request body size field, specified in kilobytes. This setting controls the overall request size limit, excluding any file uploads. Configuring an appropriate value for this field is crucial for optimizing security and performance.',
+    recommended_action: 'Modify application gateway WAF policy, enable request body inspection and set the desired request body size.',
```

Comment:
```suggestion
    recommended_action: 'Modify application gateway WAF policy and set the max body size to desired value.',
```
This makes more sense, I think